File: DemoServer/AdditionalLanguages/python/queries/queryByDocumentId/queryByDocumentId.py
from demo_example import Example, RunParamsBase
from models import Employee

class RunParams(RunParamsBase):
    def __init__(self, employee_document_id: str):
        self.employee_document_id = employee_document_id

class QueryByDocumentId(Example):
    def run(self, run_params: RunParams) -> Employee:
        employeeDocumentID = run_params.employee_document_id
        #region Demo
        with self.document_store_holder.store().open_session() as session:
            
            #region Step_1
            query_by_document_id = session.query(object_type=Employee)
            #endregion
            #region Step_2
            query_by_document_id.where_equals("Id", employeeDocumentID)
            #endregion

            #region Step_3
            employee = query_by_document_id.first()
            #endregion
        #endregion

        return employee

File: DemoServer/AdditionalLanguages/php/queries/queryByDocumentId/QueryByDocumentId.php
<?php

namespace RavenDB\Demo\queries\queryByDocumentId;

//region Usings
//endregion

use RavenDB\Demo\common\DocumentStoreHolder;
use RavenDB\Demo\common\models\Employee;

class QueryByDocumentId
{
    public function __invoke(RunParams $runParams): ?Employee
    {
        $employeeDocumentId = $runParams->getEmployeeDocumentId();
        //region Demo
        $employee = null;

        $session = DocumentStoreHolder::getStore()->openSession();
        
        try {
        //region Step_1
        $queryByDocumentId = $session->query(Employee::class)
            //endregion
            //region Step_2
                ->whereEquals("id", $employeeDocumentId);
            //endregion

            //region Step_3
            $employee = $queryByDocumentId->firstOrDefault();
            //endregion
            
        } finally {
            $session->close();
        }
        //endregion

        return $employee;
    }
}
